rtengine/iplocallab.cc

```diff
@@ -13080,7 +13080,7 @@ void ImProcFunctions::Lab_Local(
 
     //avoidcolshi(lp, sp, transformed, reserved,  cy, cx, sk);
 
-    const float radius = lp.rad / (sk * 1.4); //0 to 70 ==> see skip
+    const double radius = lp.rad / (sk * 1.4); //0 to 70 ==> see skip
```

Comment:
`radius` is cast into a `double` in several places. Those casts can be removed. There's also a few places where `radius` is used where a `float` is expected. There should be casts there.

Reply:
I spent some time trying to make correct convertion in all the places to make all the warnings disappear, but it's too complicated for now. These convertrtions double to float and vice versa are happen so often and mixed for most variables.

As for my taste, I would replace all the floats to double (or created a typealias and use it everythere to have ability to choose float/double if it's needed)

I decided to revert my changes for this particular file for now